src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String newBallColor = new ColorSupplier().getRandomColor();
+        int newBallNumber = new Random().nextInt(100);
```

Comment:
`Random` and `ColorSupplier` should be a class level fields, plus naming is incorrect, there variables are not color and number yet, they are a services which provide us with some values, and should be named properly